xrtpy/response/channel.py

```diff
@@ -240,116 +222,94 @@ def __init__(self, index, filter_number):
 
     @property
-    @u.quantity_input
-    def density(self) -> u.g * u.cm ** -3:
-        """XRT  focal plane filter density."""
-        return u.Quantity(self._fp_filter_data["DENS"], u.g * u.cm ** -3)
+    def filter_material(self) -> str:
+        """XRT filter material."""
+        return self._fp_filter_data["MATERIAL"]
 
     @property
-    @u.quantity_input
-    def wavelength(self) -> u.angstrom:
-        """Array of wavelength for every X-ray focal plane filter in angstroms."""
-        return u.Quantity(self._fp_filter_data["WAVE"], u.angstrom)[
-            : self.number_of_wavelengths
-        ]
+    def filter_mesh_transmission(self):
+        """Mesh transmission for the focal plane filter."""
+        return self._fp_filter_data["MESH_TRANS"]
 
     @property
-    def transmission(self):
-        """Get focal plane filter transmission."""
-        return self._fp_filter_data["TRANS"][: self.number_of_wavelengths]
+    def filter_name(self) -> str:
+        """XRT filter name."""
+        return self._fp_filter_data["LONG_NAME"]
 
     @property
     def number_of_wavelengths(self):
         """Data number length."""
         return self._fp_filter_data["LENGTH"]
 
     @property
-    def mesh_trans(self):
-        """Mesh transmission for the focal plane filter."""
-        return self._fp_filter_data["MESH_TRANS"]
+    def filter_substrate(self) -> str:
+        """XRT filter substrate."""
+        return self._fp_filter_data["SUBSTRATE"]
 
     @property
-    def substrate(self) -> str:
-        """XRT substrate for the focal plane filter."""
-        return self._fp_filter_data["SUBSTRATE"]
+    @u.quantity_input
+    def filter_thickness(self) -> u.angstrom:
+        """Filter thickness."""
+        return u.Quantity(self._fp_filter_data["THICK"], u.angstrom)
 
+    @property
+    def filter_transmission(self):
+        """Filter transmission."""
+        return self._fp_filter_data["TRANS"][: self.number_of_wavelengths]
 
     @property
-    def thickness(self):
-        """Filter thickness."""
-        return self._fp_filter_data["THICK"]
+    @u.quantity_input
+    def filter_wavelength(self) -> u.angstrom:
+        """XRT filter wavelength in angstroms."""
+        return u.Quantity(self._fp_filter_data["WAVE"], u.angstrom)[
+            : self.number_of_wavelengths
+        ]
+
 
 class CCD:
-    """Charge-coupled device on board XRT."""
+    """Charge-coupled device (CCD) on board X-Ray Telescope (XRT) on the Hinode spacecraft."""
 
     _genx_file = _genx_file
 
     def __init__(self, index):
         self._channel_index = index
         self._ccd_data = self._genx_file[self._channel_index]["CCD"]
 
-    @property
-    def ccd_name(
-        self,
-    ) -> str:
-        """Hinode/XRT flight model CCD."""
-        return self._ccd_data["LONG_NAME"]
-
     @property
     @u.quantity_input
-    def ccd_ev_ore_electron(
-        self,
-    ) -> u.eV / u.electron:
+    def ccd_ev_pre_electron(self) -> u.eV / u.electron:
```

Comment:
```suggestion
    def ccd_energy_per_electron(self) -> u.eV / u.electron:
```
...would need to be changed elsewhere too